tests/openapi/run.sh

```diff
@@ -43,11 +43,7 @@ function test_source() {
 }
 
 function run() {
-	if [[ $(pip list) == *"requests"* ]]; then
-		echo "requests already installed"
-	else
-		sudo pip install requests # in ci need sudo to install requests
-	fi
+	make install_test_python_dep
```

Comment:
not sure if we should pass path of Makefile to `make`, because it looks for current directory

Reply:
in `make integration_test CASE="xxx"`, current  directory is the project's root directory , so it's ok to use `make xxx` here